examples/quarkus/testsuite/pom.xml

```diff
@@ -21,5 +21,11 @@
                 <module>native</module>
             </modules>
         </profile>
+        <profile>
+            <id>blazebit-release</id>
+            <modules>
+                <module>native</module>
```

Comment:
You forgot to also add the module `base`. Specifying modules here overrides the modules AFAIK.